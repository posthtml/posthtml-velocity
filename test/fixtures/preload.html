<!doctype html>
<html>
  <head>
    <title>title</title>
    <link rel="stylesheet" href="style.css">
  </head>
  <script src="script-async.js" async></script>
  <script src="script.js"></script>
</html>
