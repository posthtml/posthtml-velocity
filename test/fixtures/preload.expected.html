<!doctype html>
<html>
  <head>
    <title>title</title>
    <link rel="preload" href="style.css" as="style" onload="this.rel='stylesheet'">
  <link rel="preload" href="script.js" as="script"><noscript><link rel="stylesheet" href="style.css"></noscript></head>
  <script src="script-async.js" async=""></script>
  <script src="script.js"></script>
</html>
